### DISCUSSION TITLE
Can I define a custom source or folder from which another site hosted on the team rep, for GitHub pages can load from?

### DISCUSSION THREAD

**Discussion by 1natalie** [leightonavantgardehaus/.github#1] [Q&A] [answered]
Instead of the normal options as the source for my site deployed with pages, that is the master branch or the docs folder in the master branch, can I define another source, like a folder called site or pages or some custom name.

**Comment by TL8N** [comment, marked as answer]
the only three options that GitHub Pages recognizes are:

master branch
docs/ folder on the master branch
gh-pages branch
So it doesn’t look like you’re able to define a custom folder or branch yet.

I Emailed you a a link to locate
